src/GraphQL/Validation/Rules/5.8 - Variables/2. VariablesAreInputTypes.cs

```diff
@@ -26,7 +28,7 @@ public class VariablesAreInputTypes : IValidationRule
         {
             var type = varDef.Type.NamedGraphTypeFromType(context.Schema);
 
-            if (!type.IsInputType())
+            if (type == null || !type.IsInputType())
```

Comment:
`IsInputType` extension method actually can handle `null` from `this` but I'm fine with this change.